{"name": "测试数据：【 测试test，True，0，删除用户信息成功 】", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'get'", "trace": "self = <testcases.api_test.test_05_delete_user.TestUserDelete object at 0x1021aa3c8>, login_fixture = {'code': 1003, 'msg': '用户名不存在！！！'}\nusername = '测试test', except_result = True, except_code = 0, except_msg = '删除用户信息成功'\n\n    @allure.story(\"用例--删除用户信息\")\n    @allure.description(\"该用例是针对获取用户删除接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{except_result}，{except_code}，{except_msg} 】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, except_result, except_code, except_msg\",\n                             api_data[\"test_delete_user\"])\n    @pytest.mark.usefixtures(\"insert_delete_user\")\n    def test_delete_user(self, login_fixture, username, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        user_info = login_fixture\n>       admin_user = user_info.get(\"login_info\").get(\"username\")\nE       AttributeError: 'NoneType' object has no attribute 'get'\n\ntestcases/api_test/test_05_delete_user.py:36: AttributeError"}, "description": "该用例是针对获取用户删除接口的测试", "attachments": [{"name": "log", "source": "0dda9a5b-74e7-44bf-8ca6-29c64df3dc38-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "58f2f486-8461-48b4-ad62-b19f2a0ff72d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'测试test'"}, {"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "'删除用户信息成功'"}], "start": 1632655255075, "stop": 1632655255076, "uuid": "0ba5905b-5956-49d6-a5ab-54b25cb0a103", "historyId": "da319e4398ddda3c7eb597b8bd4ec2fc", "testCaseId": "f140003f8969cec64326a77ad093297e", "fullName": "testcases.api_test.test_05_delete_user.TestUserDelete#test_delete_user", "labels": [{"name": "feature", "value": "用户删除模块"}, {"name": "story", "value": "用例--删除用户信息"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_05_delete_user"}, {"name": "subSuite", "value": "TestUserDelete"}, {"name": "host", "value": "txxdembp"}, {"name": "thread", "value": "11698-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_05_delete_user"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}